src/beta.ts

```diff
@@ -0,0 +1,16 @@
+// Beta/experimental features - API subject to change
```

Comment:
Nit/Question: Is `experimental` a better name? `groq-js/experimnetal` versus `groq-js/beta`? 🤔

Reply:
Renamed to experimental and re-export `parse` from there so library users only need to integrate to one entrypoint. Also solves problems with incompatible types from the different entrypoints